data-structures/ds-cque/java/CircularQueue.java

```diff
@@ -0,0 +1,126 @@
+public class CircularQueue
+{
+  int SIZE = 5;
+  int front, rear;
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6358132)